packages/markdown/src/components/VoidElements/VoidElementsWarning.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useVoidElements } from './useVoidElements';
+import { Note, Paragraph, TextLink } from '@contentful/forma-36-react-components';
+import tokens from '@contentful/forma-36-tokens';
+
+import { css } from 'emotion';
+
+const styles = {
+  note: css({
+    margin: `${tokens.spacingM} 0`
+  }),
+  paragraph: css({
+    '&:not(:last-child)': {
+      paddingBottom: tokens.spacingS
+    }
+  })
+};
+
+export function VoidElementsWarning() {
+  const { isEmpty } = useVoidElements();
+
+  if (isEmpty) {
+    return null;
+  }
+
+  return (
+    <Note noteType="warning" hasCloseButton={false} className={styles.note} testId="markdown-void-elements-warning">
+      <Paragraph className={styles.paragraph}>
+        Your markdown contains <TextLink linkType="primary" icon="ExternalLink" iconPosition="right">void
+        elements</TextLink> that causes the preview to skip those parts. This indicates incorrect usage of one or more
```

Comment:
```suggestion
        elements</TextLink> that cause the preview to skip those parts. This indicates incorrect usage of one or more
```

Reply:
Sure? :)

Reply:
Why does my brain do the Germans-learning-English-fart:

> He, she, it - das s muss mit.

As it's multiple elements (3rd person plural) that cause something, I'd argue it's without the "s", or do we have some edge-case grammar here? 🤔

Reply:
DeepL says I'm wrong 😆